SingularityUI/app/components/common/input/DateEntry.cjsx

```diff
@@ -0,0 +1,46 @@
+FormField = require './FormField'
+
+DateEntry = React.createClass
+
+    componentDidMount: ->
+        id = '#' + @props.id
+        changeFn = @props.updateFn
+        $ -> $(id).datetimepicker({
+                sideBySide: true
+                format: "ddd MMM DD YYYY HH:mm:ss [UTC]ZZ"
```

Comment:
This actually already existed as a constant, I just forgot about it. Pulling it out.